src/overlays/actors/ovl_En_Tanron5/z_en_tanron5.c

```diff
@@ -18,10 +33,15 @@ void EnTanron5_Destroy(Actor* thisx, PlayState* play);
 void EnTanron5_Update(Actor* thisx, PlayState* play2);
 void EnTanron5_Draw(Actor* thisx, PlayState* play);
 
-void func_80BE5818(Actor* thisx, PlayState* play2);
-void func_80BE5C10(Actor* thisx, PlayState* play);
+void EnTanron5_RuinFragmentItemDrop_Update(Actor* thisx, PlayState* play2);
+void EnTanron5_ItemDrop_Draw(Actor* thisx, PlayState* play);
 
-s32 D_80BE5D80 = 0;
+typedef enum {
+    /* 0 */ EN_TANRON5_ITEM_DROP_TYPE_10_ARROWS,
+    /* 1 */ EN_TANRON5_ITEM_DROP_TYPE_LARGE_MAGIC
+} EnTanron5ItemDropType;
+
+s32 ruinFragmentItemDropCount = 0;
```

Comment:
```suggestion
s32 sRuinFragmentItemDropCount = 0;
```
Along with the variable in `variables.txt`